server/admin_management/api/app/services/user_service.py

```diff
@@ -29,13 +29,12 @@ def find_or_create(self, user_type_code: str, user_name: str, requester: str):
 
         fam_user = self.get_user_by_domain_and_name(user_type_code, user_name)
         if not fam_user:
-            request_user = schemas.FamUser(
-                **{
```

Comment:
Sorry, I forgot what the "**" do in Python, do you remember and why do we remove it here?

Reply:
I think it's a way to unpack a dictionary so can pass its key-value pairs to the schema. Because I removed the FamUser schema, and create a new one called FamUserCreate with just what we need for creating a user, and no need to do this anymore.

Reply:
Although I don't feel it is due to your change on a new FamUserCreate class but it is interesting to see it simply works and probably it already worked like that before, the "**" wasn't needed maybe.... good to try later...